src/plugins/console/public/application/containers/embeddable/embeddable_console.tsx

```diff
@@ -31,8 +33,10 @@ import * as store from '../../stores/embeddable_console';
 import { setLoadFromParameter, removeLoadFromParameter } from '../../lib/load_from';
 
 import './_index.scss';
+import { EmbeddedConsoleResizeButton, getCurrentConsoleMaxSize } from './console_resize_button';
 
 const KBN_BODY_CONSOLE_CLASS = 'kbnBody--hasEmbeddableConsole';
+const CONSOLE_HEIGHT_STORAGE_KEY = 'sense:embeddedConsoleHeight';
```

Comment:
There is a storage service that gets the prefix on creation, so that in the code you only need to keep the actual key value. Here is an example in the settings https://github.com/elastic/kibana/blob/main/src/plugins/console/public/services/settings.ts. It also allows you to specify a default value when retrieving data from storage

Reply:
Yes, I meant to come back to this. I'll look into using the storage service, I will say I don't want to default the value here since the default is dynamic based on the screen size.

Although a good refactor for this later would be to move from a `px` size for the console height to using `vh` instead. Then we could use a constant default. But that would requiring doing more math on the mouse move resize to convert the `px` change to `vh` change.

Reply:
Updated the embedded console service to have functions for saving and retrieving the console height using the storage service.